backend/src/main/java/com/stampcrush/backend/application/manager/owner/ManagerCommandService.java

```diff
@@ -0,0 +1,44 @@
+package com.stampcrush.backend.application.manager.owner;
+
+import com.stampcrush.backend.application.manager.owner.dto.OwnerCreateDto;
+import com.stampcrush.backend.application.manager.owner.dto.OwnerLoginDto;
+import com.stampcrush.backend.auth.api.response.AuthTokensResponse;
+import com.stampcrush.backend.auth.application.util.AuthTokensGenerator;
+import com.stampcrush.backend.entity.user.Owner;
+import com.stampcrush.backend.exception.LoginIdBadRequestException;
+import com.stampcrush.backend.exception.OwnerNotFoundException;
+import com.stampcrush.backend.repository.user.OwnerRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@RequiredArgsConstructor
+@Transactional
+@Service
+public class ManagerCommandService {
+
+    private final OwnerRepository ownerRepository;
+    private final AuthTokensGenerator authTokensGenerator;
+
+    public Long register(OwnerCreateDto ownerCreateDto) {
+        String loginId = ownerCreateDto.getLoginId();
+        String password = ownerCreateDto.getPassword();
+
+        if (isDuplicatedLoginId(loginId)) {
+            throw new LoginIdBadRequestException("중복되는 아이디 입니다.");
+        }
+        Owner owner = Owner.of(loginId, password);
+        return ownerRepository.save(owner).getId();
+    }
+
+    private boolean isDuplicatedLoginId(String loginId) {
+        return ownerRepository.findByLoginId(loginId).isPresent();
+    }
+
+    public AuthTokensResponse login(OwnerLoginDto ownerLoginDto) {
```

Comment:
now our login method only use select query.
so we can use @Transactional(readonly = true).

how do you think?

Reply:
sorry for interrupting.
i removed the readonly annotation.
me and jena are making refresh token logic which contains insert query for black list table. 
thus we are going to remove readonly few minutes later(maybe tommorr....)
to prevent human error i think we better remove it earlier in case we do forget removing it later haha

Reply:
합의 결과 빼기로 했습니다..

Reply:
cant understand... pls speak in english please oppa